src/Microsoft.TemplateEngine.Cli/TableOutput/TemplateGroupDisplay.cs

```diff
@@ -75,20 +73,14 @@ internal static IReadOnlyList<TemplateGroupTableRow> GetTemplateGroupsForListDis
                     shortName = highestPrecedenceTemplate.Info.ShortName;
                 }
 
-                string templateType = string.Empty;
-                if (highestPrecedenceTemplate.Info.Tags != null && highestPrecedenceTemplate.Info.Tags.TryGetValue("type", out ICacheTag typeTag))
-                {
-                    templateType = typeTag.DefaultValue;
-                }
-
                 TemplateGroupTableRow groupDisplayInfo = new TemplateGroupTableRow()
                 {
                     Name = highestPrecedenceTemplate.Info.Name,
                     ShortName = shortName,
                     Languages = string.Join(",", languageForDisplay),
                     Classifications = highestPrecedenceTemplate.Info.Classifications != null ? string.Join("/", highestPrecedenceTemplate.Info.Classifications) : null,
                     Author = highestPrecedenceTemplate.Info.Author,
-                    Type = templateType
+                    Type = highestPrecedenceTemplate.Info.GetTemplateType() ?? string.Empty
                 };
```

Comment:
Before this change, if `type` was not found it was set to `typeTag.DefaultValue` now its `string.Empty`, is this intentional?

Reply:
Related to all other comments as well: tags defined in tags section of template.json are converted to Tags entry as following:
- tag name as the Key
- value as single entry in Choices and Descriptions without description
- value also as default value 

Intention of extension methods was also to unify the way how we retrieve the value. I used single to have a better chance to spot if smth goes wrong when reading the template.json/ someone modifies that code.